channels/slack/src/main/kotlin/com/justai/jaicf/channel/slack/SlackChannel.kt

```diff
@@ -38,10 +40,21 @@ class SlackChannel private constructor(
     private constructor(botApi: BotApi, urlPrefix: String): this(botApi) {
         val config = SlackConfig().apply {
             methodsEndpointUrlPrefix = urlPrefix
+            methodsConfig = MethodsConfig().apply {
+                isStatsEnabled = false
+            }
         }
 
         val slack = Slack.getInstance(config)
-        app = App(AppConfig.builder().slack(slack).build())
+        app = App(AppConfig.builder()
+            .slack(slack)
+            .alwaysRequestUserTokenNeeded(false)
+            .singleTeamBotToken("empty")
+            .signingSecret("empty")
+            .build(),
+            listOf(IgnoringSelfEvents(config))
```

Comment:
Why not to add this list to the first constructor?

Reply:
Because this filter is provided by default in AppConfig.